gradle.properties

```diff
@@ -1,12 +1,12 @@
 projectVersion=4.0.1-SNAPSHOT
 micronautDocsVersion=1.0.25
-micronautVersion=2.1.3
+micronautVersion=2.3.2
```

Comment:
I had to update this because micronaut-cache was requesting an older version of lettuce.

Reply:
2.3.4 is now current latest version, could you please try that one?

Reply:
sure! https://github.com/micronaut-projects/micronaut-redis/pull/165/commits/2bb10866d8cca5fc15db4036832503b81bef4c0a